config.json

```diff
@@ -14,7 +14,7 @@
     "solution": [".meta/example.f90"],
     "test": ["%{snake_slug}_test.f90"],
     "example": ["%{snake_slug}.f90"],
-    "exemplar": ["%{snake_slug}.f90"]
+    "exemplar": ["exemplar.f90"]
```

Comment:
This change seems unrelated to the PR

Reply:
Yes, you are right, but it came up as the linter has changed and I thought an extra PR was overkill... :-/